arda/arda.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+from sklearn.ensemble import RandomForestRegressor
+
+
+# algo 2
+# A                     the (normalized) data matrix
+# n                     the amount of features to generate
+#
+# Return: A matrix of generated random features, where each column represents one feature
+def gen_features(A, n):
+    L = []
+    m = np.mean(A, axis=1)
+    s = np.cov(A)
+    for i in range(n):
```

Comment:
This is wrong. You need to generate n*d features, where d is the number of features in A (without Y).